test/fixtures/comprehensions/assignment-expr/actual.js

```diff
@@ -0,0 +1,3 @@
+[for idx i in Array(10):
+  now x = f(i)
```

Comment:
I expect we can change the output of `now` to look like that very easily, but for some reason `addImplicitReturns` extracts the LHS of the assignment operator when returning it.

AFAIK it's perfectly fine to `return x=3` or some such, but if you had some reason for returning the LHS in `addImplicitReturns` it would be good to know why that's there, and/or why this should be the same and/or different.

Right now it's just using the same code path.